strapi/strapi
# [DEITS] Clean up transfer command

### What does it do?

Some minor fixes to the transfer command. `to` and `from` will always be treated as URLs, and if not provided a local Strapi will be used.

In the future we will add a different option to specify providers.

### Why is it needed?

Transfer wasn't really implemented properly and was only created for testing purposes before.

### How to test it?

Run `strapi transfer --to {url}` and transfer should start
Run `strapi transfer` and see that an error is given.
